docs/pt/developer-docs/hit-counts.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contagem de Acertos
+description: Aprenda a usar Contagem de Acertos no Editor de Conquistas para criar condições que precisam ser cumpridas múltiplas vezes antes de engatilhar uma conquista.
+---
+
+# Contagem de Acertos
+
+No Editor de Conquistas, o campo no lado mais à direita é a Contagem de Acertos.
+
+![image](https://user-images.githubusercontent.com/32706333/48969526-38774c80-efbd-11e8-98ec-9d6c16dba022.png)
```

Comment:
I know this is a bit out of scope, but this image needs to be fixed.

While we're in here, would you be willing to add this image to the repo itself (similar to other images in the public directory)?

Reply:
Certainly! Should I do that for all the other images like this one that I bump into?